Assignment2/MMN44-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": {
+    "vocab": "http:www.example.org/ontology#",
+    "Class01": "vocab:class",
+    "Sensor029": "vocab:sensor",
+    "Measurement8401": "vocab:measurement",
+    "Computer101": "vocab:computer",
+    "User10A": "vocab:user",
+    "prop": "http://www.example.org/properties#",
+    "includes":"prop:includes",
+    "hasMeasurement": "prop:hasMeasurement",
+    "hasTemperature": {
+      "@id": "prop:hasTemperature",
+      "@type": "xsd:integer"
+    },
+    "atTime": {
+      "@id": "prop:atTime",
+      "@type": "xsd:dateTimeStamp"
+    },
+    "hasOwner": "prop:hasOwner",
+    "hasName": {
+      "@id": "prop:hasName",
+      "@type": "xsd:string"
+    },
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  
```

Comment:
Add an id an type for each resource

Reply:
"Class01":, "Measurement8401":, "Sensor029":, "Computer101":, and "User10A": are resources, thus please provide an id and type for them between the corresponding curly braces. They aren't attributes, thus you don't need to define as such. 

Remember the representation of the WiFi point example. It has id and type because it is a resource that needs to be identified by machines. In this case, all the resources that I listed before are resources as well. 

Please, review this file carefully before resubmitting it.